README.rst

```diff
@@ -66,6 +66,12 @@ database tables and perform the initial migration ::
     flask db upgrade
     flask run
 
+To add Slack integration, set the ``SLACK_WEBHOOK`` and ``SLACK_CHANNEL``
+environmental variables ::
```

Comment:
This probably doesn't belong in `Quickstart`. Can we move this to its own section?